roles/dns/tasks/main.yml

```diff
@@ -1,4 +1,8 @@
 ---
+- name: Install prerequisites
+  package:
+    name: python-dns
```

Comment:
This package is called "python2-dns" on Fedora, so running it there would fail the playbook.